README.md

```diff
@@ -54,7 +54,7 @@ What does the seed give you out of the box?
 |demo/tests| The tests for your plugin|
 |src| The plugin source code|
 |src/platform/android| Plugin Android specific configuration|
-|src/platform/ios|Plugin ios specific configuration|
+|src/platform/ios|Plugin ios specific configuration. All configurations that are applicable via XCode Info tab and are saved in application's Info.plist file can also be applied for plugins. |
```

Comment:
It seems that some people are not aware that all changes made in the Xcode Info tab are actually doable by directly changing the `Info.plist` file. There are many guides out there having instructions like:
 > Open your Xcode project -> Go to Info Tab -> Change a configuration here

which makes people think that this can't be achieved by {N} plugin because it requires touching the Xcode project. However, it can be achieved by manually applying the needed configurations in your pluign's `Info.plist` file. The [instructions for Google Drive SDK for iOS](https://developers.google.com/drive/v3/web/quickstart/ios#step_2_prepare_the_workspace) is such an example:

 > ... With the project node still selected, select "QuickstartApp" in the TARGETS section, then select the Info tab, and expand the URL Types section. Click the + button, and add a URL scheme for your reversed client ID....


 I can't find clearly stated in the linked article that all configurations made in the Xcode Info tab are also doable through manually changing the `Info.plist` file, so I think it is worth mentioning it in a single sentence.

However, my opinion is not strong, so I leave it up to you to decide if we actually need mentioning it.

P.S.: I can help with this, so ping me if you need an input from me.